.prospector.yaml

```diff
@@ -5,3 +5,11 @@ ignore-paths:
 pycodestyle:
   options:
     max-line-length: 142
+
+pylint:
+  disable:
+    - django-not-configured
+
+pyflakes:
+  disable:
+    - F401
```

Comment:
I had to add this here to ignore `F401`. not sure why `pyflakes` raises the below error even though we have `# noqa` on line 32

```
axes/apps.py
    Line: 32
      pyflakes: F401 / 'axes.checks' imported but unused (col 9)
```